src/Interop.fs

```diff
@@ -40,26 +40,14 @@ module Interop =
             | _ -> value.ToString() |> ViewBuilder.escape
         KeyValue (key, result)
 
-    let inline mkStyle (key: string) (value: obj) : IStyleAttribute = Style (key, value) :> _
-
-type FunctionComponent<'Props> = 'Props -> ReactElement
+    let mkEventHandler(name: string) (handler: obj -> unit) : IReactProperty =
+        let event = EventHandlerType.Event(handler)
+        IReactProperty.KeyValue(name, event)
 
-// fsharplint:disable
+    let mkEventHandlerWithKey name key (handler: obj -> unit) =
+        let keyEvent = EventHandlerType.KeyEvent(key, handler)
+        IReactProperty.KeyValue(name, keyEvent)
 
-type Event () = class end
```

Comment:
removed such types as not used / useful ? event handlers are now `obj --> unit` to avoid issues with F# casting / type inference in function types (variance?)